### PR TITLE
Various minor windows-related bug fixes

### PR DESCRIPTION
**Pull request by marcusnaslund**
Fixes five parts of #1466

### CODE REVIEW

**Comment by marcusnaslund** [comment]
Peer review @sebastianbaginski ?

File: test/concurrent/WaitLockTest.ooc [hunk 1]
```diff
@@ -15,7 +15,7 @@ WaitLockTest: class extends Fixture {
 		super("WaitLock")
 		this add("_testWithMutexOwnership", This _testWithMutexOwnership)
 		this add("_testWithoutMutexOwnership", This _testWithoutMutexOwnership)
-		this add("_testWakeWithFailingCondition", This _testWakeWithFailingCondition)
+		version (!windows) { this add("_testWakeWithFailingCondition", This _testWakeWithFailingCondition) }
```
Comment (sebastianbaginski) [inline test/concurrent/WaitLockTest.ooc]: Yeah I forgot to tell you, can you add `android` here as well while you're at it ? thanks :)

**Comment by marcusnaslund** [comment]
Updated. Removed third WaitLock test from Android as well.

**Comment by sebastianbaginski** [comment]
OK great, PR done.

File: test/net/NetTest.ooc [hunk 2]
```diff
@@ -68,7 +69,7 @@ NetTest: class extends Fixture {
 				buffer := serverSocket receive(128)
 				expect(buffer toString(), is equal to(expected toString()))
 			}
-	
+
```
Comment (marcusnaslund) [inline test/net/NetTest.ooc]: Check your VSCode settings, @fredrikbryntesson

**Pull request merged**